public/css/docs.css

```diff
@@ -1269,3 +1247,542 @@ footer .container {
 
 
 /*-------------------------/.MARGIN/PADDING-------------------------*/
+
+html {
+    font-size: 100%;
+    -ms-text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%
+}
+
+body {
+    margin: 0;
+    padding: 0
+}
+
+pre,
+code,
+address,
+caption,
+th,
+figcaption {
+    font-size: 1em;
+    font-weight: normal;
+    font-style: normal
+}
+
+fieldset,
+iframe,
+img {
+    border: none
+}
+
+caption,
+th {
+    text-align: left
+}
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0
+}
+
+article,
+aside,
+footer,
+header,
+nav,
+main,
+section,
+summary,
+details,
+hgroup,
+figure,
+figcaption {
+    display: block
+}
+
+audio,
+canvas,
+video,
+progress {
+    display: inline-block;
+    vertical-align: baseline
+}
+
+button {
+    font: inherit;
+    vertical-align: middle
+}
+
+*,
+*:before,
+*:after {
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%
+}
+
+body {
+    font-family: "Helvetica Neue", Helvetica, Arial, Verdana, sans-serif;
+    font-size: 18px;
+    line-height: 1.5;
+    color: #333;
+    font-weight: 300
+}
+
+body pre,
+body code {
+    font-family: Menlo, Consolas, Monaco, "Courier New", monospace, serif
+}
+
+p {
+    margin-bottom: 1.5em;
+}
+
+h1 {
+    margin-bottom: 0.5em;
+    font-size: 3em;
+    font-weight: 200;
+    line-height: 1;
+    padding-top: 1em;
+}
+
+h1.active+.main-nav {
+    border-top: 1px solid #333
+}
+
+h2 {
+    margin-bottom: 0.5em;
+    font-size: 2.5em;
+    font-weight: 200;
+    line-height: 1;
+    padding-top: 1em;
+}
+
+h3 {
+    margin-bottom: 0.5em;
+    font-size: 1.5em;
+    font-weight: 400;
+    line-height: 1
+}
+
+h4 {
+    margin-bottom: 0.5em;
+    font-size: 1.25em;
+    font-weight: 300;
+    line-height: 1.2
+}
+
+h5 {
+    margin-bottom: 0.5em;
+    font-size: 1.175em;
+    font-weight: 500;
+    line-height: 1.4
+}
+
+h6 {
+    margin-bottom: 0.5em;
+    font-size: 1em;
+    font-weight: 700;
+    line-height: 1.5
+}
+
+pre {
+    font-size: 14px;
+    line-height: 18px;
+    border-left: 3px #ef5138 solid;
+    margin: 0.5em;
+    margin-left: 10px;
+    margin-bottom: 1.5em;
+    padding: 0.5em;
+    padding-left: 1em;
+    overflow: scroll
+}
+
+p>code,
+li>code,
+dd>code,
+blockquote>code,
+td>code {
+    color: #000;
+    padding: 3px 8px;
+    font-size: 14px;
+    white-space: nowrap;
+    border: 1px solid #E5E5E5;
+    background-color: #f7f7f7
+}
+
+hr {
+    border: none;
+    border-top: 1px #f3f3f3 solid;
+    margin: 2em 0
+}
+
+hr:last-child {
+    display: none
+}
+
+details {
+    margin-bottom: 2em
+}
+
+details:first-child {
+    margin-top: 1.5em
+}
+
+cite {
+    display: block
+}
+
+cite:before {
```

Comment:
I didn't see this being used anywhere in the current content. Am I missing something?